bin/plotting/pycbc_ifar_catalog

```diff
@@ -239,6 +244,7 @@ if opts.open_box:
     pylab.xlim(30 * min(fore_ifar), 3 * max_ifar)
 else:
     pylab.ylim(0.7, max(expected_cumnum))
+    pylab.xlim(min(expected_ifar) / 30, 3 * max(upper[4]))
```

Comment:
I still haven't quite worked out why this setting for xmin is what we want. I guess 'expected_ifar' is just a guess anyway in terms of starting at 1e-4 (about 1/3 of a day) so if we want to allow for even less significant events we might want a smaller number .. ?

Reply:
Removed - I don't know why but I thought Id reverted to the original already